ocaml/xapi/xapi_globs.ml

```diff
@@ -917,6 +917,8 @@ let winbind_machine_pwd_timeout = ref (7 * 24 * 3600)
 
 let winbind_update_closest_kdc_interval = ref (3600. *. 24.) (* every day *)
 
+let winbind_kerberos_encryption_type = ref "strong"
```

Comment:
In practice this is a default value. I don't like the default string being in two different modules. Please use a variant type (like before), or find a way to define it only once and refer to that binding from modules that use it.